app/kumactl/cmd/apply/apply_test.go

```diff
@@ -178,6 +183,37 @@ var _ = Describe("kumactl apply", func() {
 		Expect(resource.Meta.GetNamespace()).To(Equal("default"))
 	})
 
+	It("should apply a new Dataplane resource from URL", func() {
+		// setup http server
+		mux := http.NewServeMux()
+		server := httptest.NewServer(mux)
+		port, err := strconv.Atoi(strings.Split(server.Listener.Addr().String(), ":")[1])
+		Expect(err).ToNot(HaveOccurred())
+
+		go func() {
+			defer GinkgoRecover()
+			// when
+			http.Handle("/testdata/", http.StripPrefix("/testdata/", http.FileServer(http.Dir("./testdata"))))
+			log.Fatal(http.ListenAndServe(fmt.Sprintf(":%v", port), nil))
```

Comment:
Tips why CI is probably failing:
Drop this line, it's not needed with `httptest`.
call `Handle` or `HandleFunc` on mux.
Goroutine is no longer needed.